src/components/ModalAppUpdate.js

```diff
@@ -90,86 +95,86 @@ export default class ModalAppUpdate extends Component {
 
     onShowUnderlayCancelButton = () => {
       this.setState({ pressStatusCancel: true });
-    }   
-    
+    }
+
     onHideUnderlayConfirmButton = () => {
       this.setState({ pressStatusConfirm: false });
     }
 
     onShowUnderlayConfirmButton = () => {
       this.setState({ pressStatusConfirm: true });
-    }   
+    }
 
     render() {
         const { isFocused } = this.state;
 
-        const component1 = () => 
+        const component1 = () =>
         <Image
           source={img1}
         />
-    
-        const component2 = () =>     
+
+        const component2 = () =>
         <Image
         source={img2}
         />
-    
-        const component3 = () => 
+
+        const component3 = () =>
         <Image
         source={img3}
         />
-    
-        const component4 = () =>                 
+
+        const component4 = () =>
         <Image
         source={img4}
         />
-    
-        const component5 = () =>                 
+
+        const component5 = () =>
         <Image
         source={img5}
         />
-    
-        const component6 = () =>                 
+
+        const component6 = () =>
         <Image
         source={img6}
         />
-    
+
         const buttonsOne = [{ element: component1}, { element: component2, id: 2 }, { element: component3, id: 3 }];
         const buttonsTwo = [{ element: component4, id: 4 }, { element: component5, id: 5 }, { element: component6, id: 6 }];
-                
+
         return (
           <View>
             <View style={styles.content}>
-                <View style={styles.textContent}> 
-                    <Text style={styles.text}>Profile Picture</Text> 
+                <View style={styles.textContent}>
+                    <Text style={styles.text}>Profile Picture</Text>
                 </View>
                 <ButtonGroup
                   selectedIndex={this.state.selectedIndexRowOne}
                   buttons={buttonsOne}
                   onPress={this.updateIndexOne}
                   containerStyle={{height: 70}}
-                />   
+                />
                 <ButtonGroup
                   selectedIndex={this.state.selectedIndexRowTwo}
                   buttons={buttonsTwo}
                   onPress={this.updateIndexTwo}
                   containerStyle={{height: 70}}
-                />    
+                />
 
                 <View style={styles.username}>
-                    <TextInput 
+                    <TextInput
                       style={{flex: 1}}
-                      type="TextInput" 
-                      name="myTextInput" 
-                      placeholder='Change your username' 
+                      type="TextInput"
+                      name="myTextInput"
+                      placeholder= {this.props.currentUsername}
```

Comment:
if they haven't set a username the default should be something like as previous (Change your username)

Reply:
This is in the update modal, when they open it for the first time the initial modal will show